azure-mgmt-resources/src/test/java/com/azure/management/resources/core/TestBase.java

```diff
@@ -212,6 +214,7 @@ public void write(int b) {
                     .withHttpClient(generateHttpClientWithProxy(null))
                     .withHttpLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BODY_AND_HEADERS))
                     .withPolicy(new ResourceGroupTaggingPolicy())
+                    .withPolicy(new TimeoutPolicy(Duration.ofMinutes(1)))
```

Comment:
Need consider several cases for timeout: 
- Hot is working on the Polling case? 
- How is it working with the server side retry logic header (429) ?

Reply:
Yes, so it is just added for unit test currently, since parallel test will cause timeout usually.
In my opinion,
* The polling case could remain the same, but add logic for cancel if there are lots of time out.
* The 429 handler should be added before timeout in the product situation.